
original script File:
    "/home/feng/figureScript/1119__peakBW__141CS15__CAMTA.py"
<br/>
Link to directory
<a href="/static/figures/1119__peakBW__141CS15__CAMTA">
    OUTPUT directory
</a>
<br/>

Success: True
    
<br/>
Link to figure:
<a href="/static/figures/1119__peakBW__141CS15__CAMTA/figure.html">
    figure.html
</a>

<br/>
<a href="/static/figures/1119__peakBW__141CS15__CAMTA/1119__peakBW__141CS15__CAMTA.log">
runLog
</a>
:
    <pre>
"/home/feng/envs/server/bin/python
/home/feng/figureScript
/home/feng/envs/server/lib/python2.7
/home/feng/envs/server/lib/python2.7/plat-x86_64-linux-gnu
/home/feng/envs/server/lib/python2.7/lib-tk
/home/feng/envs/server/lib/python2.7/lib-old
/home/feng/envs/server/lib/python2.7/lib-dynload
/usr/lib/python2.7
/usr/lib/python2.7/plat-x86_64-linux-gnu
/usr/lib/python2.7/lib-tk
/home/feng/envs/server/local/lib/python2.7/site-packages
/home/feng/envs/server/lib/python2.7/site-packagesis in ipython: 0 
[WARN]No module named jinja2_util
is in ipython: 0 
[WARN]No module named jinja2_util
is in ipython: 0 
[WARN]No module named jinja2_util
is in ipython: 0 
[WARN]No module named jinja2_util
[WARN] pymisca.vis_util cannot find network
[WARN] pymisca.vis_util cannot find network
[WARN] Cannot find file:key.gene
[WARN] Cannot find file:key.gene
[WARN] pymisca.vis_util cannot find network
[WARN] pymisca.vis_util cannot find network
/home/feng/envs/CAMTA/per_FC-GT-0dot9_141CS15_ICE1.narrowPeak
[L] 4785
[L]= 100
[L]= 100
[L]= 100
[L]= 100
[L]= 100
/home/feng/envs/server/local/lib/python2.7/site-packages/synotil/CountMatrix.py:142: UserWarning: Pandas doesn&#39;t allow columns to be created via a new attribute name - see https://pandas.pydata.org/pandas-docs/stable/indexing.html#attribute-access
  self.param = {&#39;normF&#39;:&#39;identityNorm&#39;,
/home/feng/envs/server/local/lib/python2.7/site-packages/synotil/CountMatrix.py:401: UserWarning: Pandas doesn&#39;t allow columns to be created via a new attribute name - see https://pandas.pydata.org/pandas-docs/stable/indexing.html#attribute-access
  self.summary = df
/home/feng/envs/server/local/lib/python2.7/site-packages/pymisca/vis_util.py:686: VisibleDeprecationWarning: Passing `normed=True` on non-uniform bins has always been broken, and computes neither the probability density function nor the probability mass function. The result is only correct if the bins are uniform, when density=True will produce the same result anyway. The argument will be removed in a future version of numpy.
  ys,edg = np.histogram(xs,bins,normed=normed)

 [WARN] unable to simplify condName. Exception:list index out of range
DataAcc
141CS14          141C_S14_1283-MYC-22C-SD-ZT6
141CS15    141C_S15_1283-MYC-22C-4C-shift-ZT6
194CS4                 194C_S4_CAMTA3-123-22C
194CS5             194C_S5_CAMTA3-123-22C-CHX
194CS6                  194C_S6_CAMTA3-123-4C
Name: header, dtype: object
Index([u&#39;141C_S14_1283-MYC-22C-SD-ZT6&#39;, u&#39;141C_S15_1283-MYC-22C-4C-shift-ZT6&#39;,
       u&#39;194C_S4_CAMTA3-123-22C&#39;, u&#39;194C_S5_CAMTA3-123-22C-CHX&#39;,
       u&#39;194C_S6_CAMTA3-123-4C&#39;],
      dtype=&#39;object&#39;, name=u&#39;bwFile&#39;)
[&#39;141C_S14_1283-MYC-22C-SD-ZT6&#39; &#39;141C_S15_1283-MYC-22C-4C-shift-ZT6&#39;]
(&#39;[OUTPUT]&#39;, &#39;/home/feng/envs/testserver/static/figures/1119__peakBW__141CS15__CAMTA/figure.html&#39;)
"
    <pre/>
    